node-launchpad/src/bin/tui/main.rs

```diff
@@ -21,11 +21,12 @@ use node_launchpad::{
 #[cfg(target_os = "windows")]
 use sn_node_manager::config::is_running_as_root;
 use sn_peers_acquisition::PeersArgs;
+use sn_protocol::version::IDENTIFY_PROTOCOL_STR;
 use std::{env, path::PathBuf};
 use tokio::task::LocalSet;
 
 #[derive(Parser, Debug)]
-#[command(author, version = version(), about)]
+#[command(author, version = version(), about, name = "Autonomi Node Launchpad")]
```

Comment:
The `fn version()` is from the init of the node launchpad crate and it displays the author + data dir path, which is incorrect. We can modify it to dispaly the correct versions.

But I'm not sure `protocol version` applies to launchpad as it can use any safenode version with any protocol version within. So, maybe we can display just the `CARGO_PKG_VERSION` here?